apps/emqx/src/emqx_misc.erl

```diff
@@ -556,6 +556,60 @@ try_to_existing_atom(Convert, Data, Encoding) ->
         _:Reason -> {error, Reason}
     end.
 
+is_sensitive_key(token) -> true;
```

Comment:
hmm. i though we have ported this function long ago. there are more keys in 4.4 ?

Reply:
Yes, but some are not usually used, here are only common ones, and then the redact/2 allow to pass a parameter as a custom checker function